### PR TITLE
Che4z-1.0.0

### PR DESCRIPTION
**Pull request by filipkroupa**
Signed-off-by: Filip Kroupa <filip.kroupa@broadcom.com>

### What does this PR do?
Make Che4z plugins referenced in Che4z devfile available in plugin registry

[https://github.com/eclipse/che-devfile-registry/pull/147](https://github.com/eclipse/che-devfile-registry/pull/147)
[https://www.eclipse.org/che/docs/che-7/eclipse-che4z/](https://www.eclipse.org/che/docs/che-7/eclipse-che4z/)

### CODE REVIEW

File: v3/plugins/broadcommfd/cobol-language-support/0.8.3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+type: VS Code extension
+displayName: COBOL Language Support
+title: COBOL Language Support
+description: COBOL Language Support standardizes the communication between language tooling and your code editor using the Language Server Protocol (LSP)
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/eclipse/che-che4z-lsp-for-cobol
+category: Language
+spec:
+  containers:
+    - image: eclipse/che-remote-plugin-runner-java8:7.3.0
```
Comment (amisevsk) [inline v3/plugins/broadcommfd/cobol-language-support/0.8.3/meta.yaml]: For non-release branches of Che, plugins should use the `next` tag of plugin runners. `7.3.0` is already 3 weeks old, and `7.4.0` has been released.

File: v3/plugins/bitlang/cobol/5.0.6/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+type: VS Code extension
+displayName: COBOL
+title: COBOL
+description: COBOL, JCL, PL/I & DIR syntax support
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/spgennard/vscode_cobol
+category: Language
+spec:
+  extensions:
+    - https://github.com/eclipse/che-che4z/releases/download/1.0.0/cobol-5.0.6.vsix
```
Comment (amisevsk) [inline v3/plugins/bitlang/cobol/5.0.6/meta.yaml]: Please ensure all files end in newlines.

Comment (filipkroupa) [reply v3/plugins/bitlang/cobol/5.0.6/meta.yaml]: Added missing newlines

File: v3/plugins/broadcommfd/cobol-language-support/0.8.3/meta.yaml (hunk 1 above)
Comment (filipkroupa) [reply v3/plugins/broadcommfd/cobol-language-support/0.8.3/meta.yaml]: We would prefer to release with version we built with and tested against. We experienced some issues during testing if we did not provide this version tag. Of course if the `next` tag is necessary, we can retest.

Comment (amisevsk) [reply v3/plugins/broadcommfd/cobol-language-support/0.8.3/meta.yaml]: Fair enough -- makes sense to me.

**Comment by filipkroupa** [comment]
We are targeting [Milestone 7.5.0](https://github.com/eclipse/che/milestone/105) Can we expect this to be part of the release?

**Comment by amisevsk** [comment]
> We are targeting Milestone 7.5.0 Can we expect this to be part of the release?

Merged, this PR will be included in 7.5.0

**Pull request merged**